starter_code/elevator.js

```diff
@@ -3,17 +3,85 @@ class Elevator {
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = 'Up';
+    this.waitingList;
+    this.passengers = [];
+    this.timer;
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
+  start() { 
+    this.timer = setInterval(() => {
+      this.update();
+    }, 1000);
+  }
+
+  stop() {
+    clearInterval(this.timer);
+   }
+
+// Uodate function should  display 
+// The current status of the elevator by calling the log function. 
+  update() {
+    if (this.direction ==='Up'){
+      this.floorUp();
+    }
+    if (this.direction === 'Down'){
+      this.floorDown();
+    }
+    
```

Comment:
Have to call the `this._passengersEnter();` at some point, might want to call it here.